app/src/main/java/com/xmartlabs/scasas/doapp/controller/GroupController.java

```diff
@@ -0,0 +1,49 @@
+package com.xmartlabs.scasas.doapp.controller;
+
+import com.raizlabs.android.dbflow.sql.language.SQLite;
+import com.xmartlabs.scasas.doapp.model.Group;
+import com.xmartlabs.scasas.doapp.model.Group_Table;
+import com.xmartlabs.scasas.doapp.model.User;
+
+import java.util.List;
+
+import rx.Single;
+import rx.android.schedulers.AndroidSchedulers;
+import rx.schedulers.Schedulers;
+
+/**
+ * Created by santiago on 3/21/17.
+ */
+public class GroupController extends Controller {
+    public Single<Group> insertGroup(Group group) {
```

Comment:
CheckResult enforces the caller to use the result of this call, i.e., you cannot just say:
```java
groupController.insertGroup(..);
```
but rather, you must do something with it, like:
```java
groupController.insertGroup(...).subscribe();
```